include/spectrum.h

```diff
@@ -334,6 +334,10 @@ extern uint  __LIB__    zx_screenstr_callee(uchar row, uchar col) __smallc __z88
 #define zx_setcursorpos(a,b)      fputc_cons(22); fputc_cons(a); fputc_cons(b);
 #define zx_topleft()              fputc_cons(22); fputc_cons(0); fputc_cons(0);
 
+// Print a string on the screen
+extern int printk(char *fmt, ...);
```

Comment:
This would need to be `extern int __LIB__ printk(char *fmt)`

I think it's probably better to just do this using a library alias:

```
extern int zx_printf(char *fmt,...)
```

And then in the library create two new files somewhere in libsrc/target/zx/ that looks like this

SCCZ80:
```
PUBLIC zx_printf
EXTERN printk
defc zx_printf = printk
```

ZSDCC:
```
PUBLIC _zx_printf
EXTERN _printk
defc _zx_printf = _printk
```

Reply:
Okay, I modified `spectrum.h` and created `zx_printf.asm` in `libsrc/target/zx`, with the contents for _sccz80_ you wrote above.
But... how do I create the _ZSDCC_ version?

Reply:
Just create _zx_printf.asm and copy the ZSDCC version into that.

If you haven't already you'll need to add reference to both files to libsrc/target/zx/spectrum.lst